specification/electronic-prescription-service-api.yaml

```diff
@@ -320,6 +305,64 @@ paths:
                     A successful response to a prescription-order, dispense-notification or dispense-claim request.
                   value:
                     $ref: components/examples/operation-outcome-success.json
+        '4XX':
+          description: |
+            An error occurred as follows:
+
+            | HTTP status | Error code  | Description      |
+            | ----------- | ----------- | ---------------- |
+            | 400         | BAD_REQUEST | Invalid request. |
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+              examples:
+                OperationOutcome:
+                  description: |
+                    An error response.
+                  value:
+                    $ref: components/examples/example-a-validation-error-missing-field/Response-FhirError.json
+  /FHIR/R4/$process-message#prescription-order-update:
+    post:
+      operationId: send-prescription-order-update-message
+      summary: Cancel a prescription
+      description: |
+        ## Overview
+        Use this endpoint to send a prescription-order-update message to EPS.
+
+        The effect of calling this endpoint depends on the type of the message, which is determined by the `MessageHeader` resource.
+        To cancel a prescription, send a [prescription-order-update message](https://simplifier.net/guide/DigitalMedicines/prescription-order-update).
+
+        Note that the `#prescription-order-update` in the URL has been added to distinguish this from the other operations which
+        can be performed using the `$process-message` endpoint. It is not required in requests to the API.
+      tags:
+        - prescribing
+      parameters:
+        - $ref: '#/components/parameters/BearerAuthorization'
+        - $ref: '#/components/parameters/RequestID'
+        - $ref: '#/components/parameters/CorrelationID'
+      requestBody:
+        required: true
+        content:
+          #FIXME - should be set to application/fhir+json but we have had to change it due to an Apigee SmartDocs bug.
```

Comment:
line 316 has this bug 'fixed', wonder if its worth removing these instead of adding more?

Reply:
Done